pkg/process/checks/process_rt.go

```diff
@@ -119,25 +118,25 @@ func fmtProcessStats(
 	chunked := make([][]*model.ProcessStat, 0)
 	chunk := make([]*model.ProcessStat, 0, cfg.MaxPerMessage)
 
-	for _, fp := range procs {
+	for pid, fp := range procs {
 		// Skipping any processes that didn't exist in the previous run.
 		// This means short-lived processes (<2s) will never be captured.
```

Comment:
Nit: this should be `<10s`, right?

Reply:
Ah right.

Reply:
Well it is comparing with `lastProcs` which is generated 2s ago. In the context of whole running process-agent you are right, but in the context of rt_process alone it's still 2s.

Reply:
Oh yeah, that makes sense. Thanks for explaining!